airflow/utils/operator_helpers.py

```diff
@@ -199,6 +199,8 @@ def determine_kwargs(
 
 def make_kwargs_callable(func: Callable[..., R]) -> Callable[..., R]:
     """
+    # TODO: D205-ify this one
```

Comment:
Advice?

Reply:
How about "Creates a new callable that only forwards necessary arguments from any provided input."